src/platform/serializers.js

```diff
@@ -0,0 +1,151 @@
+    currency: rawAccount.currency,
+    balance: new BigNumber(rawAccount.balance),
+    spendableBalance: new BigNumber(rawAccount.spendableBalance),
+    blockHeight: rawAccount.blockHeight,
+    lastSyncDate: new Date(rawAccount.lastSyncDate),
+  };
+}
+
+export function serializePlatformEthereumTransaction(
+  transaction: PlatformEthereumTransaction
+): RawPlatformEthereumTransaction {
+  return {
+    family: transaction.family,
+    amount: transaction.amount.toString(),
+    recipient: transaction.recipient,
+    nonce: transaction.nonce,
+    data: transaction.data ? transaction.data.toString() : undefined,
+    gasPrice: transaction.gasPrice
+      ? transaction.gasPrice.toString()
+      : undefined,
+    gasLimit: transaction.gasLimit
+      ? transaction.gasLimit.toString()
+      : undefined,
+  };
+}
+
+export function deserializePlatformEthereumTransaction(
+  rawTransaction: RawPlatformEthereumTransaction
+): PlatformEthereumTransaction {
+  return {
+    family: rawTransaction.family,
+    amount: new BigNumber(rawTransaction.amount),
+    recipient: rawTransaction.recipient,
+    nonce: rawTransaction.nonce,
+    data: rawTransaction.data ? Buffer.from(rawTransaction.data) : undefined,
+    gasPrice: rawTransaction.gasPrice
+      ? new BigNumber(rawTransaction.gasPrice)
+      : undefined,
+    gasLimit: rawTransaction.gasLimit
+      ? new BigNumber(rawTransaction.gasLimit)
+      : undefined,
+  };
+}
+
+export function serializePlatformBitcoinTransaction(
+  transaction: PlatformBitcoinTransaction
+): RawPlatformBitcoinTransaction {
+  return {
+    family: transaction.family,
+    amount: transaction.amount.toString(),
+    recipient: transaction.recipient,
+    feePerByte: transaction.feePerByte
+      ? transaction.feePerByte.toString()
+      : undefined,
+  };
+}
+
+export function deserializePlatformBitcoinTransaction(
+  rawTransaction: RawPlatformBitcoinTransaction
+): PlatformBitcoinTransaction {
+  return {
+    family: rawTransaction.family,
+    amount: new BigNumber(rawTransaction.amount),
+    recipient: rawTransaction.recipient,
+    feePerByte: rawTransaction.feePerByte
+      ? new BigNumber(rawTransaction.feePerByte)
+      : undefined,
+  };
+}
+
+export function serializePlatformTransaction(
+  transaction: PlatformTransaction
+): RawPlatformTransaction {
+  switch (transaction.family) {
+    case "ethereum":
+      return serializePlatformEthereumTransaction(transaction);
+    case "bitcoin":
+      return serializePlatformBitcoinTransaction(transaction);
+    default:
+      throw new Error(`Can't serialize ${transaction.family} transactions`);
+  }
+}
+
+export function deserializePlatformTransaction(
+  rawTransaction: RawPlatformTransaction
+): PlatformTransaction {
+  switch (rawTransaction.family) {
+    case "ethereum":
+      return deserializePlatformEthereumTransaction(rawTransaction);
+    case "bitcoin":
+      return deserializePlatformBitcoinTransaction(rawTransaction);
```

Comment:
note for future: in future we want to split these into modules ethereum.js , bitcoin.js etc..
actually even better is to have a file `src/families/*/platform.js` that export the coin specifics.